fabric-content-registries-v0/src/main/resources/fabric-content-registries-v0.mixins.json

```
@@ -9,7 +9,8 @@
     "MixinAbstractFurnaceBlockEntity",
     "MixinFireBlock",
     "OxidizableMixin",
-    "ShovelItemAccessor"
+    "ShovelItemAccessor",
+    "SculkSensorBlockMixin"
```

```suggestion
    "SculkSensorBlockMixin",
    "ShovelItemAccessor"
```
Alphabetical ordering